mis_builder/models/aep.py

```diff
@@ -328,7 +328,7 @@ def get_aml_domain_for_dates(self, date_from, date_to,
         return expression.normalize_domain(domain)
 
     def do_queries(self, date_from, date_to, period_from, period_to,
-                   target_move):
+                   target_move, additional_move_line_filter=[]):
```

Comment:
No, but this means that any variable initalized to the same value (`[]`) in any method will be shared, so you must change it.